client/src/components/select-formula-view/StyledBox.js

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import Box from '@mui/material/Box';
+import styled from 'styled-components';
+
+const StyledBox = styled(Box)`
+width: 53vw;
+min-width:  400px;
+max-width: 1000px;
```

Comment:
Indentions of 2 spaces could be added here even though the linter doesn't spot them.